src/main/java/com/staxrt/tutorial/model/Post.java

```diff
@@ -0,0 +1,80 @@
+package com.staxrt.tutorial.model;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "posts")
+@EntityListeners(AuditingEntityListener.class)
+public class Post {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+
+    @Column(name = "data", nullable = false)
+    private String data;
+
+    @Column(name = "privacy", nullable = false)
+    private String privacy;
+
+
+    @CreationTimestamp
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "created_at", nullable = false)
+    private Date createdAt;
+
+    @CreatedBy
+    @Column(name = "created_by", nullable = false)
+    private String createdBy;
+
+    @UpdateTimestamp
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "updated_at", nullable = false)
+    private Date updatedAt;
+
+    @LastModifiedBy
+    @Column(name = "updated_by", nullable = false)
+    private String updatedBy;
+
+    public long getId() { return id; }
```

Comment:
Zamiast pisać gettery/settery/toString może byś spróbował użyć https://projectlombok.org/ ? Ewentualnie do rozważenia użycie java 14: https://www.baeldung.com/java-record-keyword tylko wtedy musiałbyś napisać w dev guide wymaganie że trzeba mieć jdk 14.

Reply:
Obyczaje te cudenko, zawsze się pisało sety gety to już z automatu się robi

Reply:
Tym bardziej że idea automatycznie mi je tworzy

Reply:
obczaj, bo developerzy są przewrażliwieni jak ktoś je jeszcze ręcznie tworzy. oba koncepty bardzo łatwo się włącza i warto znać

Reply:
Done